foldamers/utilities/util.py

```diff
@@ -203,8 +203,7 @@ def distances(interaction_list, positions):
     distance_list = []
 
     for interaction in interaction_list:
-        if interaction[0] < len(positions) and interaction[1] < len(positions):
-            distance_list.append(distance(positions[interaction[0]], positions[interaction[1]]))
+        distance_list.append(distance(positions[interaction[0]], positions[interaction[1]]))
 
```

Comment:
It should be fairly easy to replace this with mdtraj.compute_distances. http://mdtraj.org/1.9.3/api/generated/mdtraj.compute_distances.html
We just need to make the pair list into an [n_pair x 2] numpy array. 

I can make that a separate issue though.